### DISCUSSION TITLE
VS Code: Poor contrast on selected text

### DISCUSSION THREAD

**Discussion by aquacash5** [daltonmenezes/aura-theme#106] [General]
It is very hard to see what text is selected on any theme.

I am using `Aura Dark`
![2022-02-18 14_34_01-Window](https://user-images.githubusercontent.com/10753165/154703372-bcd1a446-7ae3-4027-aa17-61fb164473cc.png)

**Comment by daltonmenezes** [comment]
Hi @aquacash5 , I believe the text selection can't be stronger, because  some texts colorized by the syntax highlight can disappear in relation to the selection background.

**Reply by daltonmenezes** [reply]
@aquacash5 you can try to override it manually by using a color that fits better for you like:
```js
"workbench.colorCustomizations": {
  "editor.selectionBackground": "#3d375e7f",
}
```

**Reply by aquacash5** [reply]
Thank you,

I find this works well for me

```
"workbench.colorCustomizations": {
    "[Aura Dark]": {
      "editor.selectionBackground": "#a277ff5f"
    }
  }
```
![2022-02-18 15_26_11-Window](https://user-images.githubusercontent.com/10753165/154711706-849b44a3-ea91-4cf4-a012-a3d9007aa518.png)
